backend/app/core/config.py

```
@@ -0,0 +1,40 @@
+from pydantic_settings import BaseSettings
+from typing import List, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)